docs/source/tutorial/ptb.rst

```diff
@@ -0,0 +1,293 @@
+The Implementation of Recurrent Neural Net Language Model with Chainer
```

Comment:
"with Chainer" is obvious, so please omit it. Then how about "Write an RNN Language Model" for the title?